src/actions/transactionEstimateActions.js

```diff
@@ -106,22 +106,32 @@ export const estimateTransactionsAction = (transactionsToEstimate: TransactionTo
 
     const activeAccountAddress = getAccountAddress(activeAccount);
 
-    const transactions = await Promise.all(transactionsToEstimate.map(({
-      to,
-      data,
-      value,
-      assetData,
-    }) => buildEthereumTransaction(
-      to,
-      activeAccountAddress,
-      data,
-      Number(value).toString(),
-      assetData?.token,
-      assetData?.decimals,
-      assetData?.tokenType,
-      assetData?.contractAddress,
-      assetData?.id,
-    )));
+    let transactions;
+    try {
+      transactions = await Promise.all(transactionsToEstimate.map(({
+        to,
+        data,
+        value,
+        assetData,
+      }) => buildEthereumTransaction(
+        to,
```

Comment:
Thanks, imo it won't change much in this case, just it would require more code to add in order to construct an object here. Will leave as it is, hope it makes sense 👍